sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -37,7 +37,7 @@ extensions:
         pkg_name="origin"
         echo $pkg_name > PKG_NAME
         echo "openshift-ansible openshift-ansible-callback-plugins openshift-ansible-docs openshift-ansible-filter-plugins openshift-ansible-lookup-plugins openshift-ansible-playbooks openshift-ansible-roles" > OPENSHIFT_ANSIBLE_PKGS
-        sudo python sjb/hack/determine_install_upgrade_version.py $( cat OPENSHIFT_ANSIBLE_BUILT_VERSION ) > OPENSHIFT_ANSIBLE_VARS
+        sudo python sjb/hack/determine_install_upgrade_version.py $( cat OPENSHIFT_ANSIBLE_BUILT_VERSION ) --dependency_branch ${ORIGIN_TARGET_BRANCH} > OPENSHIFT_ANSIBLE_VARS
```

Comment:
How come isn't this affected by https://github.com/openshift/aos-cd-jobs/issues/315?